src/main/scala/com/vngrs/etl/transformers/RowToJsonTransform.scala

```diff
@@ -0,0 +1,28 @@
+package com.vngrs.etl.transformers
+
+import com.vngrs.etl.Transform
+import org.apache.spark.rdd.RDD
+import org.apache.spark.sql.{Row, SQLContext}
+
+/**
+  * Transform [[org.apache.spark.sql.Row]]s to `JSON` formatted [[String]]s.
+  */
+final case class RowToJsonTransform() extends Transform[Row, String] {
+
+  /**
+    * Transform [[org.apache.spark.sql.Row]]s to `JSON` formatted [[String]]s.
+    *
+    * @param input [[org.apache.spark.rdd.RDD]] of [[org.apache.spark.sql.Row]]s
+    * @return Transformed [[org.apache.spark.rdd.RDD]] of `JSON` [[String]]s
+    */
+  override def apply(input: RDD[Row]): RDD[String] = {
+    val sqlCtx = new SQLContext(input.sparkContext)
+
+    val df = input.take(1).headOption.map(_.schema) match {
+      case Some(schema) => sqlCtx.createDataFrame(input, schema)
+      case None         => sqlCtx.emptyDataFrame
```

Comment:
I am against to throwing exceptions. They are not idiomatic in scala and there are better ways to handle errors.

BTW is there any other way to convert rows to JSON without knowing its schema?